src/aws/policy.ts

```diff
@@ -15,10 +15,14 @@ export default class Policy {
     this.name = new Name(options)
   }
 
-  toJSON(): Record<string, unknown> {
+  toJSON(): Record<string, any> {
```

Comment:
Can you disable eslint no-explicit-any on this next line?

`// eslint-disable-next-line @typescript-eslint/no-explicit-any`